### PR TITLE
Update wrong configuration

### PR DESCRIPTION
**Pull request by mwaeckerlin**
Please update the configuration bugfix to the homepage.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -123,7 +123,11 @@ GEM
       polyglot
       polyglot (>= 0.3.1)
     tzinfo (0.3.37)
+<<<<<<< HEAD
```
Comment (tpreusse) [inline Gemfile.lock:126]: needs conflict resolving

**Comment by mwaeckerlin** [comment]
I'm not interested in Gemfile.lock. And I did not change it. The computer obviously did, not me. I don't care.

What's that?

Isn't that even a file that is overwritten at runtime anyway and should not be checked in? Lockfiles shouldn't be checked in. Generated files are normally not checked in.

If I remove Gemfile.lock, then restart the server, everything works fine and the Gemfile.lock is recreated. So for me it is clear that Gemfile.lock should be removed from git. It is not needed and created automatically if you run the server.

Importand to me is

**Comment by tpreusse** [comment]
Don't worry about it. I already published the proposed changes.

It's good practice to check the Gemfile.lock, it ensures that the same gem
versions are used in production.

What must have happened is that you did a merge and had a conflict in the
Gemfile.lock. However there is a special case: no JS runtime -> don't check
in your Gemfile.lock - prod servers have a JS runtime.

**Comment by mwaeckerlin** [comment]
Ok, thank you.

Yes, I merged your changes.

**Pull request closed without merging**